portfolio/src/main/java/com/google/sps/Puzzle.java

```diff
@@ -26,6 +26,17 @@
 
   private String name = "";
 
+  private String username = "";
+
+  public Puzzle create(Long puzzleId, String imageUrl, Difficulty difficulty, String name, String username) {
```

Comment:
I would recommend removing this create method because it is a bit confusing. For example initializing the puzzleId to -1 when creating a new puzzle is a code smell.